silhouette-crypto-jca/build.sbt

```diff
@@ -1,13 +0,0 @@
-import Dependencies._
```

Comment:
Just curious, why we had an `sbt` file for the dependencies? Where is this useful? 🤔

Reply:
To avoid duplication when it is referenced. It is a common practice in Scala projects.
```
libraryDependencies ++= Seq(
        Library.commonsLang3,
        Library.Play.cache,
        ...
```